### DISCUSSION TITLE
Logs / prevent bruteforce attack

### DISCUSSION THREAD

**Discussion by pstraebler** [karakeep-app/karakeep#321] [Q&A] [answered]
Hi,

I'm starting to use Hoarder, which I really like, and I'm planning to make it publicly accessible on my server.
Before doing so, I was wondering how to prevent myself from being attacked by Bruteforc? I'd like to use Fail2ban for example, but I don't see any log files generated by Hoarder. Docker logs obviously don't contain connection attempts.
I know there was a 'LOG_LEVEL' variable, but it doesn't seem to have any effect.
Any ideas?

Cheers !
Pierre

**Comment by MohamedBassem** [comment, marked as answer]
Hi, indeed hoarder doesn't right now log its http requests. What you can do is put a reverse proxy (e.g. nginx) in front of hoarder and use its logs for fail2ban. Putting Hoarder behind a reverse proxy is a good idea anyways if you're going to be exposing publicly (e.g. for https, etc)

**Comment by pstraebler** [comment]
I'm not used to filtering hosts to ban via apache logs, but this can indeed do the trick.
Thanks!
